plugin/keyring_vault/vault_keyring.cc

```diff
@@ -148,6 +149,10 @@ static int keyring_vault_init(MYSQL_PLUGIN plugin_info)
         " file. Please also make sure Vault is running and accessible."
         " The keyring_vault will stay unusable until correct configuration file gets"
         " provided.");
+
+      if (current_thd != NULL)
+        push_warning(current_thd, Sql_condition::SL_WARNING, 42000,
```

Comment:
Magic number `42000`. Should it be `ER_SOMETHING`?

Reply:
This is a number used, when there is no appropriate ER_SOMETHING to be used